presto-main/src/main/java/io/prestosql/testing/MaterializedResult.java

```diff
@@ -382,7 +381,7 @@ else if (prestoValue instanceof SqlTimeWithTimeZone) {
                         zone);
             }
             else if (prestoValue instanceof SqlTimestamp) {
-                convertedValue = SqlTimestamp.JSON_FORMATTER.parse(prestoValue.toString(), LocalDateTime::from);
+                convertedValue = ((SqlTimestamp) prestoValue).toLocalDateTime();
```

Comment:
Please see https://github.com/prestosql/presto/pull/4570 and https://github.com/prestosql/presto/pull/4570/commits/5bce33ec71c0641e863564ca68da755909a5c0c3

@findepi wanted to avoid rounding there to make tests more strict.

Reply:
@findepi @martint I moved these commits to a separate PR https://github.com/prestosql/presto/pull/4875